### ISSUE TITLE
ERROR - 'NegativeKeywordConflictReportRequest' object has no attribute 'Time'

### ISSUE THREAD

**Issue by animenon** [BingAds/BingAds-Python-SDK#116]
I get this error but I am not using any attribute named "Time" in the request.

Attributes used in request - AccountName','AccountNumber','AccountId','CampaignName','CampaignId','AdGroupName','AdGroupId','Keyword','KeywordId','NegativeKeyword','ConflictLevel','BidMatchType','NegativeKeywordListId','NegativeKeywordList','NegativeKeywordId','NegativeKeywordMatchType','AccountStatus','CampaignStatus','AdGroupStatus','KeywordStatus','ConflictType'

**Comment by eric-urban** [comment]
@animenon most ReportRequest objects (not the columns) have a Time attribute e.g., [here](https://github.com/BingAds/BingAds-Python-SDK/blob/master/examples/v12/report_requests.py#L285). Time is not applicable for the negative keyword conflict report. If you borrowed from one of  the samples you'll need to remove report_request.Time=time when using the [NegativeKeywordConflictReportRequest](https://docs.microsoft.com/en-us/bingads/reporting-service/negativekeywordconflictreportrequest?view=bingads-12). 

I hope this helps!

**Comment by animenon** [comment]
@eric-urban I don't use that for NegativeKeywordConflictReport but I still get this error.
Don't know what I am doing wrong here.
My code used to work great for v11. Facing this in v12.

**Comment by animenon** [comment]
@eric-urban thanks for looking into it.

Do point me to any sample code snippet of the same, if you find some..

**Comment by eric-urban** [comment]
@animenon please feel free to reopen here or via the [MSDN forum](https://social.msdn.microsoft.com/Forums/en-US/home?forum=BingAds) with your repro steps. Thanks.

**Comment by eric-urban** [comment]
@animenon I can repro the same issue. We'll need to update the Python SDK. In the meantime as a workaround you can clone the SDK repo and build a local dist. 

1. git clone https://github.com/BingAds/BingAds-Python-SDK.git

2. Edit normalize_request within reporting_service_manager.py to check whether the Time attribute exists within report_request:

```python
def normalize_request(self, report_request):

    if report_request is None:
        return

    if not hasattr(report_request, 'Time'):
        return

    if hasattr(report_request.Time, 'ReportTimeZone') \
    and hasattr(report_request.Time.ReportTimeZone, 'value') \
    and report_request.Time.ReportTimeZone.value is None:
        report_request.Time.ReportTimeZone=None
        
    if hasattr(report_request.Time, 'PredefinedTime') \
    and hasattr(report_request.Time.PredefinedTime, 'value') \
    and report_request.Time.PredefinedTime.value is None:
        report_request.Time.PredefinedTime=None
```

3. Build local SDK e.g., PS PATH_TO_LOCAL_SDK_CLONE\BingAds-Python-SDK> python.exe make.py dist

4. Upgrade with your local build e.g., (env) PS PATH_TO_LOCAL_SDK_CLONE\BingAds-Python-SDK\examples> pip install --upgrade C:\dev\github\BingAds-Python-SDK\dist\bingads-12.13.3.2.zip

I've reopened this GitHub issue. We will update the issue thread with the SDK fix / release schedule. 

@qitia FYI

**Comment by eric-urban** [comment]
@animenon this issue is resolved with release [12.13.4](https://github.com/BingAds/BingAds-Python-SDK/releases/tag/v12.13.4). Please reopen if you have any follow up questions or concerns.
